docs/db/refactor_platform_rs.dbml

```diff
@@ -81,9 +81,9 @@ Table refactor_platform.actions {
   user_id uuid [not null, note: 'User that created (owns) the action']
   due_by timestamptz
   status status [not null]
-  status_changed_at timestamptz
-  created_at timestamp [not null, default: `now()`]
-  updated_at timestamp [not null, default: `now()`]
+  status_changed_at timestamptz [not null, default: `now()`]
```

Comment:
Just to be clear, I think this will set `status_changed_at` to "now" when we first create an action. Which might be somewhat misleading for newly created actions. TBH that seems reasonable to me but want to make sure that was your intention.

Reply:
Great question to double check. Yes, this is intentional. It's because we don't have a non-state option for `Status`. If we did, I'd leave this as something we could leave unset. The way I tweaked things in this PR and the frontend one for actions is the "not_started" enum is chosen by default, so it is being set.

Thoughts?